website/docs/r/policy_set_definition.html.markdown

```diff
@@ -74,7 +76,11 @@ A `policy_definition_reference` block supports the following:
 
 * `policy_definition_id` - (Required) The ID of the policy definition or policy set definition that will be included in this policy set definition.
 
-* `parameters` - (Required) A mapping of the parameter values for the referenced policy rule. The keys are the parameter names.
+* `parameters` - (Optional / **Deprecated in favour of `parameter_values`**) A mapping of the parameter values for the referenced policy rule. The keys are the parameter names.
```

Comment:
in this instance we can remove this since the deprecated version is available

Reply:
Removed